functional-tests/src/test/resources/suites/simpleTestSuite.xml

```diff
@@ -24,7 +24,7 @@
             <!--<class name="org.eclipse.che.selenium.dashboard.CreateAndDeleteProjectsTest"/>-->
             <!--<class name="org.eclipse.che.selenium.dashboard.RenameWorkspaceTest"/>-->
             <class name="org.eclipse.che.selenium.miscellaneous.DialogAboutTest"/>
-            <!--<class name="com.redhat.che.functional.tests.WorkspaceTest"/>-->
+            <class name="com.redhat.che.functional.tests.WorkspaceTest"/>
```

Comment:
It would be nice to have changes not related to the issue you are trying to solve in different PRs.

Reply:
Well, the test is using the che-starter templates, which are being injected by the `@Workspace` annotation. It's effectively testing my changes, so I thought it wouldn't be a bad idea to enable it.